src/main/webapp/app/components/downloadButton/DownloadButton.tsx

```diff
@@ -17,7 +18,9 @@ export const DownloadButton: React.FunctionComponent<IDownloadButton> = props =>
     <a
       href={href}
       download={download || true}
-      className={`btn btn-primary ${buttonSizeClassName} ${className} ${styles.aTag}`}
+      className={`btn btn-${
+        props.outline ? 'outline-' : ''
+      }primary ${buttonSizeClassName} ${className} ${styles.aTag}`}
```

Comment:
Use `classNames()`. If any of the variables are undefined, then it will add `undefined` to class name.